ggml/src/ggml-cuda/common.cuh

```diff
@@ -269,35 +263,38 @@ static __device__ __forceinline__ half ggml_cuda_hmax(const half a, const half b
 }
 
 static __device__ __forceinline__ half2 ggml_cuda_hmax2(const half2 a, const half2 b) {
-#if !(defined(GGML_USE_HIP) && defined(__HIP_PLATFORM_AMD__))
+#if !(defined(GGML_USE_HIP) && defined(__HIP_PLATFORM_AMD__)) || (defined(GGML_USE_HIP) && HIP_VERSION >= 50700000)
```

Comment:
Wouldn't it be simpler to have something like

```
#if defined(GGML_USE_HIP) && HIP_VERSION >= 50700000
...
#elif !(defined(GGML_USE_HIP) && defined(__HIP_PLATFORM_AMD__))
...
```

Reply:
done